rotkehlchen/chain/evm/constants.py

```diff
@@ -5,6 +5,8 @@
 
 from .types import string_to_evm_address
 
+DEFAULT_TOKEN_DECIMALS: Final = 18
+
```

Comment:
nitpick: But next time when you touch a file like that and see other constants not final, please change them too. The `Final` typing concept is something new so codebase needs refreshing.